worker/sessionpool.go

```diff
@@ -97,6 +97,25 @@ func (ss *sharedSession) Revision(ctx context.Context) (rhpv2.ContractRevision,
 	return s.Revision(), nil
 }
 
+func (ss *sharedSession) RenewContract(ctx context.Context, prepareFn func(rev types.FileContractRevision, host rhpv2.HostSettings) ([]types.Transaction, types.Currency, func(), error)) (_ rhpv2.ContractRevision, _ []types.Transaction, err error) {
```

Comment:
`nit` is there a reason for the named return variables here? 

```suggestion
func (ss *sharedSession) RenewContract(ctx context.Context, prepareFn func(rev types.FileContractRevision, host rhpv2.HostSettings) ([]types.Transaction, types.Currency, func(), error)) (rhpv2.ContractRevision, []types.Transaction, error) {
```